boards/st/nucleo_l552ze_q/Kconfig.nucleo_l552ze_q

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2020 Linaro Limited
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_NUCLEO_L552ZE_Q
+	select SOC_STM32L552XX
+
+config BOARD_NUCLEO_L552ZE_Q_STM32L552XX_NS
+	bool
+	select SOC_STM32L552XX
```

Comment:
~~Not resolved~~ please amend commits in future rather than adding to the final commit with an incorrect commit

Reply:
`an incorrect commit`. Did you read commit description ?
 This is actually a dedicated refactoring commit with a proper justification.

Reply:
>  soc: stm32l5: Rename overlays for nucleo_l552ze_q ns target.
> 
> File name should now be nucleo_l552ze_q_stm32l552xx_ns.overlay

No relation to this change at all